Camera2SlowMotion/app/src/main/java/com/example/android/camera2/slowmo/fragments/CameraFragment.kt

```diff
@@ -308,7 +308,7 @@ class CameraFragment : Fragment() {
                     // Prevents screen rotation during the video recording
                     requireActivity().requestedOrientation =
                             ActivityInfo.SCREEN_ORIENTATION_LOCKED
-                    CameraActivity.output.write("started capture")
+                    CameraActivity.output.write("started capture*")
```

Comment:
`*` acts as a delimiter here. (Based on current servers script, will be improved in JSON formatting of messages)